force-app/main/default/flexipages/Deal_Support_Process_Page_Single_Related_Lists.flexipage-meta.xml

```diff
@@ -3129,16 +3139,6 @@
         <type>Region</type>
     </flexiPageRegions>
     <flexiPageRegions>
-        <itemInstances>
```

Comment:
This appears to be the activity element on the top right of this page in full1, I don't have this in my dev sandbox. But I'm not sure that it should be on the page?<img width="1272" alt="image" src="https://github.com/jnelson-prosci/prosci-sfdc/assets/169184842/c679653c-52a6-4e8e-a8ef-b468b9505dac">